### ISSUE TITLE
Bug: Lua error since 1.0.2

### ISSUE THREAD

**Issue by f4ndor** [SFX-WoW/AceGUI-3.0_SFX-Widgets#29]
**Game Version**

Retail - 8.3.7

**Add-On Version**

1.0.2

[Add-On Version]

**Describe the Issue**

Getting this lua error on login:

```lua
2x ...dOns\AceGUI-3.0-39_SFX-Widgets\Widgets\SFX-Info-URL.lua:81: attempt to concatenate field 'Escape' (a nil value)
[string "@AceGUI-3.0-39_SFX-Widgets\Widgets\SFX-Info-URL.lua"]:81: in main chunk

Locals:
Type = "SFX-Info-URL"
Version = 1
AceGUI = <table> {
 objPools = <table> {
 }
 RegisterAsContainer = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:494
 RegisterWidgetType = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:532
 WidgetBase = <table> {
 }
 RegisterAsWidget = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:510
 counts = <table> {
 }
 LayoutRegistry = <table> {
 }
 GetLayout = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:556
 GetWidgetVersion = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:586
 tooltip = AceGUITooltip {
 }
 Create = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:144
 GetWidgetCount = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:580
 GetNextWidgetNum = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:569
 WidgetRegistry = <table> {
 }
 WidgetVersions = <table> {
 }
 RegisterLayout = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:546
 Release = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:178
 ClearFocus = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:230
 WidgetContainerBase = <table> {
 }
 SetFocus = <function> defined @Ace3\AceGUI-3.0\AceGUI-3.0.lua:220
}
max = <function> defined =[C]:-1
CreateFrame = <function> defined =[C]:-1
BackdropTemplateMixin = nil
L = <table> {
 ESC = "ESC"
 Copy = "Copy"
 Press %s to copy. = "Press %s to copy."
 Click to select this text. = "Click to select this text."
 Select = "Select"
 CTRL+C = "CTRL+C"
 Press %s to cancel. = "Press %s to cancel."
}
Locale = "enUS"
KEY_COPY = "|cffffcc00CTRL+C|r"
(*temporary) = "|cffffcc00"
(*temporary) = nil
(*temporary) = "|r"
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to concatenate field 'Escape' (a nil value)"
```

**Comment by StormFX** [comment]
Sorry about that. That's what I get for doing stuff when I'm tired. Should be fixed.

**Comment by StormFX** [comment]
Fixed @ 3dfe608. Closing.
